packages/graphql_codegen/lib/builder.dart

```diff
@@ -133,7 +133,8 @@ class GraphQLBuilder extends Builder {
     AssetId targetAssetId,
     Library library,
   ) {
-    final formatter = DartFormatter();
+    final formatter = DartFormatter(
+        languageVersion: DartFormatter.latestShortStyleLanguageVersion);
```

Comment:
When Dart 3.7 arrives we may want to change it to `DartFormatter.latestLanguageVersion`